shell/screen.md

```diff
@@ -114,3 +114,14 @@ $ screen
 $ kinit
 ```
 
+{% callout "tmux" %}
+`tmux` is another program you can use to keep things running remotely. 
+In practice, it is very similar to screen, but it has some minor 
+differences. You can find syntax guides easily online, including guides 
+showing the equivalent commands in tmux and screen, but here's a quick 
+list of equivalent commands to those used in this lesson:
+* `tmux ls` instead of `screen -list`
+* `Ctrl-b d` to detach instead of `Ctrl-a d` (`tmux` in general uses 
+  `Ctrl-b` instead of `Ctrl-a`)
+* `tmux a` or `tmux attach` instead of `screen -rD`
+{% endcallout %}
```

Comment:
It would be nice to link to a guide or to here (because we might be in a better position to judge what guides are good).

Reply:
Yeah, the reason I didn't was that the guides I use seem to be pretty unofficial. [This one](http://hyperpolyglot.org/multiplexers) is quite useful, but I wasn't sure whether we wanted to link to it.

Reply:
IMO unofficial is fine. Something styled more like a blog post would be cool, but the one you posted is good as a reference.

Reply:
done. (Didn't immediately find a good blog post--stuck with the link above.)

Reply:
👍 Thanks a bunch!